AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -187,6 +187,10 @@ - (void)enqueueItem:(id<MSLog>)item flags:(MSFlags)flags {
   });
 }
 
+- (void)authTokenContext:(MSAuthTokenContext *)__unused authTokenContext didSetAuthToken:(nullable NSString *)__unused authToken {
+  [self checkPendingLogs];
```

Comment:
This would need synchronization because executed in the caller thread + check for if the channel is suspended.